lib/utils.ts

```
@@ -1,45 +1,237 @@
+export function categorizeError(errorType: string, errorMessage: string, category?: string): {
+  normalizedType: string;
+  normalizedMessage: string;
+  normalizedCategory: string;
+  originalType: string;
+  originalMessage: string;
+  originalCategory?: string;
+  groupKey: string;
+} {
+  let normalizedType = errorType;
+  let normalizedMessage = errorMessage;
+  let normalizedCategory = category || 'unknown_error';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'normalizedCategory' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)